tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract_nth.extended.txt

```diff
@@ -1,4 +1,53 @@
 REGEXP_EXTRACT_NTH("",'([a-z]+)\s+(\w+)',2)
 REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'([a-z]+)\s+(\w+)',2)
 REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'([a-z]+)\s+(\w+)',0)
-REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'',2)
\ No newline at end of file
+REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'',2)
+REGEXP_EXTRACT_NTH("abc", "(abc)", 1) //"abc"
```

Comment:
Two things:

1. `tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract_nth.extended.txt` and `tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract_nth.full.txt` have the same tests. (`tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract_nth.extended.txt` has a few more). Need to remove the repeated tests and update expecteds.
2. Revert naming change (same comment as on `tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.extract.txt`), put the new tests to the new `.full` or `.extended`.